File: TYPESCRIPT PROGRAMMING/Test54.ts
// 54. PRINT THE RANGE1 TO RANGE2 IN THR FORM OF (A^2)
const squareRange = (range1: number, range2: number): void => {
    for(let i: number = range1; i <= range2; i++) {
        process.stdout.write(`${i*i} `);
    }
    console.log();
}
squareRange(10, 20);
squareRange(30, 40);

File: JAVA PROGRAMMING/Test54.java
// 54. PRINT THE RANGE1 TO RANGE2 IN THR FORM OF (A^2)
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range1 Value: ");
        int range1 = in.nextInt();
        System.out.println("Enter Range2 Value: ");
        int range2 = in.nextInt();
        for(int i = range1; i <= range2; i++) {
            System.out.print(i * i+" ");
        }
        in.close();
    }
}